demo/demo-schema/src/main/java/org/apache/servicecomb/demo/produceprocessor/ProduceJsonProcessor.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.demo.produceprocessor;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import javax.ws.rs.core.MediaType;
+
+import org.apache.servicecomb.common.rest.codec.RestObjectMapper;
+import org.apache.servicecomb.common.rest.codec.produce.ProduceProcessor;
+
+import com.fasterxml.jackson.databind.JavaType;
+
+public class ProduceJsonProcessor implements ProduceProcessor {
```

Comment:
Does ProduceJsonProcessor copied from the formal code? Override this will cause all tests not running against formal code. I'd prefer not add this test case here.

Reply:
delete this, use override ProduceAppXmlProcessor instead, but can't gives wrong output,because we  need return correct result in springmvc demo ,we can validate it by judge whether getOrder return -1